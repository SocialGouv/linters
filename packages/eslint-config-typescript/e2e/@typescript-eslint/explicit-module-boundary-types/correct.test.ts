import {
  fixtures,
  fixtureToTestCase,
  ruleTest,
  TIMEOUT,
} from "e2e/rule_tester";
import { basename } from "path";

describe(`@typescript-eslint/${basename(__dirname)} off`, () => {
  test.each(fixtures(__dirname, /^correct$/).map(fixtureToTestCase))(
    "should validate : %s",
    (_, filePath) => ruleTest(filePath),
    TIMEOUT
  );
});
