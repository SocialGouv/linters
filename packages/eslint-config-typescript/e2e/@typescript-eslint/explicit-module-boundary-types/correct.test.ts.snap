// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@typescript-eslint/explicit-module-boundary-types correct 1`] = `
Array [
  Object {
    "errorCount": 0,
    "filePath": StringContaining "e2e/@typescript-eslint/explicit-module-boundary-types/correct.ts",
    "fixableErrorCount": 0,
    "fixableWarningCount": 0,
    "messages": Array [],
    "usedDeprecatedRules": Array [],
    "warningCount": 0,
  },
]
`;
